src/main/kotlin/com/example/moiza/domain/community/vote/service/AddVoteService.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.moiza.domain.community.vote.service
+
+import com.example.moiza.domain.community.poll.domain.repository.PollOptionRepository
+import com.example.moiza.domain.community.poll.exception.PollOptionNotFoundException
+import com.example.moiza.domain.community.vote.domain.Vote
+import com.example.moiza.domain.community.vote.domain.VoteRepository
+import com.example.moiza.domain.community.vote.exception.AlreadyVotedException
+import com.example.moiza.domain.user.facade.UserFacade
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+
+@Service
+@Transactional
+class AddVoteService(
+    private val voteRepository: VoteRepository,
+    private val pollOptionRepository: PollOptionRepository,
+    private val userFacade: UserFacade
+) {
+
+    fun execute(pollOptionId: Long) {
+        val user = userFacade.getCurrentUser()
+
+        val pollOption = pollOptionRepository.findById(pollOptionId).orElseThrow { PollOptionNotFoundException }
```

Comment:
findByIdOrNull을 사용해서 ?:으로 처리하는게 좋을 것 같아요

```suggestion
        val pollOption = pollOptionRepository.findByIdOrNull(pollOptionId)
            ?: PollOptionNotFoundException
```